### PR TITLE
fix: fixed class name in breadcrumbs_#1410

### LINKED ISSUE #1410: Fix class names in `Breadcrumb`
### Which package has an issue?

@yamada-ui/react

### Package Version

1.3.9

### Description

Corrected the following class names according to BEM

- `ui-breadcrumb__item__separator`
  - fixed to `ui-breadcrumb__separator`
- `ui-breadcrumb__item__ellipsis`
  - fixed to `ui-breadcrumb__ellipsis`

Please be careful of the following when working

Create the changeset with `major`.
Please derive a new branch from the `v2.0` branch and create a pull request for the `v2.0` branch, not the `main` branch, as we plan to release the product in v2.0.0.

### Link to Reproduction

_No response_

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Do you know how to fix the issue?

None

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by darkcat666**
Closes #1410

## Description
Fixed class name in breadcrumbs
> Add a brief description

## Current behavior (updates)

> Please describe the current behavior that you are modifying

## New behavior

> Please describe the behavior or changes this PR adds

### CODE REVIEW

**Review by illionillion — APPROVED** [review]
LGTM

File: .changeset/mean-clocks-juggle.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/breadcrumb": major
```
Comment (suzukisan22) [inline .changeset/mean-clocks-juggle.md:2]: i think this fix is not major 🤔  at least, minor change...
@illionillion 
do you think this fix deserve major??

Comment (suzukisan22) [reply .changeset/mean-clocks-juggle.md:2]: > Create the changeset with major.
Please derive a new branch from the v2.0 branch and create a pull request for the v2.0 branch, not the main branch, as we plan to release the product in v2.0.0.

sorry, i now found the line at description.
- https://github.com/yamada-ui/yamada-ui/issues/1410

**Review by suzukisan22 — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
Thanks!

**Pull request merged**